### PR TITLE
[SB-20] Add posts to user home page

### PR DESCRIPTION
**Pull request by lizamuromskaya**
image

### CODE REVIEW

**Comment by lizamuromskaya** [comment]
image

**Comment by lizamuromskaya** [comment]
image

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
-  def index; end
+  def index
+  	@posts = Post.all.page params[:page]
```
Comment (m3xq) [inline app/controllers/home_controller.rb:3]:
```suggestion
  	@posts = Post.page(params[:page])
```

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
 
+
+<div class="container">
+<% @posts.each do |post|%>
+  <div class="row mt-5">
+    <div class="col-12">
+    		<h2><%= post.name %></h2>
+    		<%= post.description.slice(0..160) + '...' %>
+    		<%= link_to "Show more", post_path(post) %> <br> 	
+    		by <%= post.created_at%>
```
Comment (B0RlS) [inline app/views/home/index.html.erb:13]: could you make a more user friendly time display here?

File: app/models/post.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 class Post < ApplicationRecord
+  paginates_per 5
```
Comment (B0RlS) [inline app/models/post.rb:2]:
```suggestion
  default_scope { order(created_at: :desc) }
```

File: app/models/post.rb [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
   belongs_to :admin_user, class_name: 'AdminUser'
 
   validates :name, presence: true, length: { minimum: 2 }
-  validates :description, presence: true, length: { minimum: 20, maximum: 500 }
+  default_scope { order(created_at: :desc) }
```
Comment (B0RlS) [inline app/models/post.rb:6]:
```suggestion
  paginates_per 5
```

Comment (B0RlS) [reply app/models/post.rb:6]: swap them

File: config/routes.rb [hunk 5]
```diff
@@ -12,4 +12,5 @@
   get 'home/index'
   get '/about', to: 'home#about'
   root 'home#index'
+ 	resources :post, only: :show
```
Comment (B0RlS) [inline config/routes.rb:15]: swap resources and root

**Pull request merged**